src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,10 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) != null) {
+            return userService.findByEmail(email).getPassword().equals(password);
```

Comment:
Should we search the user inside the database twice?

Reply:
fixed